### ISSUE TITLE
[Feature] Update helm chart and add some feature

### ISSUE THREAD

**Issue by bvis** [foomo/pagespeed_exporter#33]
Hi, it seems that the helm chart is a bit old and I'm missing some features, like the simple usage of push-gateway.

That's why I've created this repo: https://github.com/softonic/pagespeed-exporter-chart

If you find this useful I could update the provided chart here to maintain just 1 version or switch the reference in the project to my version.

By now the helm chart is available to use in a public helm hub: https://artifacthub.io/packages/helm/softonic/pagespeed-exporter

**Comment by Romiko** [comment]
Hey, I like your helm chart and I am testing it out now.

Do you have the same issue where scraping the pod directly can take over 30 seconds? So it is impractical for prometheus to scrape directly. The way around it, is only scrape via the gateway?

**Comment by Romiko** [comment]
I created a PR with your work and some minor fixes to get your chart working correctly.

https://github.com/foomo/pagespeed_exporter/pull/35

**Comment by bvis** [comment]
I've tested it just via the gateway, but you could get it working if you define a higher timeout value in the scraping config in Prometheus, I guess.

On other hand it's common to use Prometheus Operator, it provides the ServiceMonitor resource where you can define more configuration values like the timeout value and other service discovery values. If you need some help I could try to adapt my helm chart and test it.

**Comment by Romiko** [comment]
Yeah, I have it working nicely. The secret is removing the proves and HPA. I will create a new PR now for this 👍

**Comment by Romiko** [comment]
When I get a chance, I will optimize the image so http requests are immediate and it uses the previous run cycle as the results.

**Comment by bvis** [comment]
Hi, I forgot to mention you that yesterday I did the change to add the serviceMonitor, you can see it in the 1.1.0 version of the chart: https://artifacthub.io/packages/helm/softonic/pagespeed-exporter

**Comment by smartinov** [comment]
So, is this still active?
